kafka/modules/kafka/scripts/dynamic_resources.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Ok so you imported this file from upstream, using for Kafka 2.2 but not for the 2.1 as mentioned in one of the previous comments.

Reply:
fixed